src/Microsoft.DotNet.ProjectModel/Resolution/PackageDependencyProvider.cs

```diff
@@ -17,13 +17,15 @@ namespace Microsoft.DotNet.ProjectModel.Resolution
     public class PackageDependencyProvider
     {
         private readonly FallbackPackagePathResolver _packagePathResolver;
+        private readonly VersionFolderPathResolver _versionFolderPathResolver;
         private readonly FrameworkReferenceResolver _frameworkReferenceResolver;
 
         public PackageDependencyProvider(INuGetPathContext nugetPathContext, FrameworkReferenceResolver frameworkReferenceResolver)
         {
             if (nugetPathContext != null)
             {
                 _packagePathResolver = new FallbackPackagePathResolver(nugetPathContext);
+                _versionFolderPathResolver = new VersionFolderPathResolver(nugetPathContext.UserPackageFolder);
```

Comment:
Good point. I will pass a null path since this resolver is only used for building filenames.